DecipherNow/gm-fabric-dashboard
# fix: instance polling error

Resolves #1365 

## Proposed Changes

  - Added a bool `isDisabled` which is true when there is no selectedInstance. 
  - Added a tooltip to let the user know they need to select an instance
  - Added disabled styles to the polling button svg and instance slider
  
## How to Test

  - Navigate to http://localhost:3000/#/settings
  - Notice that the button is disabled and a tooltip renders on hover. The slider can still move but does not trigger any actions.
  - Go select an instance
  - Check that the button is enabled and you are able to set the polling interval as expected
